### PR TITLE
Fix for issue with custom icon on activity bar

### PR DESCRIPTION
**Pull request by daimor**
Fixes the issue raised in #8426

### CODE REVIEW

**Comment by kittaakos** [comment]
I have manually applied the changes to my branch, and it works nicely 👍 

Screen Shot 2020-09-15 at 11 35 31

I have a question, can we get rid all of the additional `50% 50% / var(--theia-private-sidebar-icon-size) no-repeat;` from the original [PR](https://github.com/eclipse-theia/theia/pull/8426/)?

**Comment by daimor** [comment]
> I have a question, can we get rid all of the additional 50% 50% / var(--theia-private-sidebar-icon-size) no-repeat; from the original PR?
Yeah, I'll take care

**Comment by daimor** [comment]
It's ready

**Comment by kittaakos** [comment]
Thank you! I am going to review it in short...

File: packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts [hunk 1]
```diff
@@ -192,8 +192,8 @@ export class PluginViewRegistry implements FrontendApplicationContribution {
         const iconClass = 'plugin-view-container-icon-' + viewContainer.id;
         const iconUrl = PluginSharedStyle.toExternalIconUrl(viewContainer.iconUrl);
         toDispose.push(this.style.insertRule('.' + containerClass + '.' + iconClass, () => `
-                mask: url('${iconUrl}') no-repeat 50% 50%;
-                -webkit-mask: url('${iconUrl}') no-repeat 50% 50%;
+                mask: url('${iconUrl}');
+                -webkit-mask: url('${iconUrl}');
```
Comment (kittaakos) [inline packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts:196]: This was not a change in the original PR.

Comment (daimor) [reply packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts:196]: Right, but it's the same and uses the same style.

Comment (kittaakos) [reply packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts:196]: I am glad you're taking care of the code that much, but I would like to ask you to revert it. This change was not included in the previous PR. Let's focus on the followings:
 - fix the regression, and
 - revert the unnecessary `50% 50% / var(--theia-private-sidebar-icon-size) no-repeat` changes from the previous PR.

Once you have reverted this, we are ready to go. Your changes work fine. Please make sure you have squashed your commits; I would like to see one single commit for the PR.

Thank you!

Comment (daimor) [reply packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts]: done

Comment (kittaakos) [reply packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts]: OK, keeping two commits actually makes sense. You're a true hero to come back and fix it so quickly.🎖️ Thank you!

**Review by kittaakos — APPROVED** [review]
Thank you for the fix, @daimor 👍

**Pull request merged**